### ISSUE TITLE
Image upload & send improvements (New Chat input)

### ISSUE THREAD

**Issue by simonricoo** [status-im/status-desktop#758]
Scope:
1. Drag and Drop functionality
2. New UI connected with the new chat input

<img width="614" alt="Screenshot 2020-08-24 at 10 21 46" src="https://user-images.githubusercontent.com/52249767/91021297-e16c3480-e5f3-11ea-9ee7-e90dddc23b1e.png">
<img width="714" alt="Screenshot 2020-08-24 at 10 23 47" src="https://user-images.githubusercontent.com/52249767/91021307-e6c97f00-e5f3-11ea-81f3-3b61b4d245a4.png">
<img width="689" alt="Screenshot 2020-08-24 at 10 24 10" src="https://user-images.githubusercontent.com/52249767/91021360-f648c800-e5f3-11ea-8192-629b78bbd831.png">

Figma file: https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=1944%3A0

**Comment by iurimatias** [comment]
to be done after #1374 (drag & drop)

**Comment by iurimatias** [comment]
already done by @emizzle as part of #1374
